bitnami/ghost/templates/deployment.yaml

```
@@ -16,6 +16,7 @@ spec:
       {{- end }}
     spec:
       {{- include "ghost.imagePullSecrets" . | indent 6 }}
+      priorityClassName: "{{ .Values.priorityClassName }}"
```

```suggestion
      {{- if .Values.priorityClassName }}
      priorityClassName: {{ .Values.priorityClassName | quote }}
      {{- end }}
```

A small change, just to follow our newest templates.